spec/provider.js

```diff
@@ -107,6 +107,30 @@ describe('BIG IoT Provider', () => {
         expect(foundOffering.spatialExtent.city).to.equal(off.extent.city);
       });
     });
+    it('should be able to delete an offering', () => {
+        expect(off.id).to.be.a('string', 'Offering ID needs to be available');
+      const query = gql`
+        query offering($offeringId:String!) {
+          offering(id:$offeringId) {
+            id name
+            rdfAnnotation { uri label }
+            spatialExtent { city }
+          }
+        }
+      `;
+      prov.delete(off)
+        .then((result) => {
+            return prov.client.query({
+            query: query,
+            variables: {
+              offeringId: result,
+            },
+              })
+              .then((result) => {
```

Comment:
I would prefer the "offering should no longer be on the marketplace" check to be a separate test case after deletion, but otherwise looks fine.